### PR TITLE
disable Xdebug on Travis CI

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ matrix:
     - php: hhvm
 
 before_install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi
```
Comment (javiereguiluz) [inline .travis.yml]: Just asking: why the `hhvm` check?

Comment (xabbuh) [inline .travis.yml]: There's no Xdebug extension on HHVM.

Comment (voronkovich) [inline .travis.yml]: @xabbuh, missing ]
BTW all shells support short syntax:
```sh
[[ "$TRAVIS_PHP_VERSION" != "hhvm" ]] && phpenv config-rm xdebug.ini
```

Comment (javiereguiluz) [inline .travis.yml]: @xabbuh yes ... let me ask differently: why not remove `xdebug.ini` always? After all, HHVM doesn't have xdebug, so the result would be the same.

Comment (xabbuh) [inline .travis.yml]: @javiereguiluz IIRC the command would fail if the file didn't exist.

Comment (javiereguiluz) [inline .travis.yml]: ok, merging then :)

**Comment by javiereguiluz** [comment]
Thanks @xabbuh! We all love fast tests :)

**Pull request closed without merging**